acoustid/future/fpindex/updater.py

```
@@ -0,0 +1,172 @@
+        async def drop_replication_slot_on_error(*exc_details) -> None:
+            if exc_details[0] is not None:
+                await drop_replication_slot(conn)
+
+        exit_stack.push_async_exit(drop_replication_slot_on_error)
+
+        total_count = await conn.fetchval(
+            """
+            SELECT reltuples::bigint AS estimate
+            FROM pg_class
+            WHERE relname = 'fingerprint'
+            """
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)